src/core/core.tooltip.js

```diff
@@ -214,6 +214,7 @@ function createTooltipItem(chart, item) {
 	return {
 		label: indexScale ? '' + indexScale.getLabelForValue(parsed[indexScale.id]) : '',
 		value: valueScale ? '' + valueScale.getLabelForValue(parsed[valueScale.id]) : '',
+		custom: parsed._custom,
```

Comment:
Instead of exposing `custom`, can we have the bar chart provide the correct value for `value`?

Reply:
Agreed, I'll do that in another request.